### PR TITLE
[Main] Updated Plugins "addon" from stable

### PR DESCRIPTION
**Pull request by OzzieIsaacs**
### Describe the changes

If updated and merged all plugins of type Addon from stable to development branch (the develop branch version was roughly 2 years old. 
I didn't not test the addons, I just merged it and tried to start pyload-ng 

### Is this related to a problem?

Many plugins are outdated and may not work in current environment

### CODE REVIEW

**Comment by vuolter** [comment]
Some imports look missing, I'll make a deep check soon and let you know.

**Comment by OzzieIsaacs** [comment]
Thank you, if I can help you, let me know. On the website, I can't see into the travis result

File: src/pyload/plugins/addons/Captcha9Kw.py [hunk 1]
```diff
@@ -49,7 +50,7 @@ class Captcha9Kw(BaseAddon):
     __description__ = """Send captchas to 9kw.eu"""
     __license__ = "GPLv3"
     __authors__ = [
-        ("RaNaN", "RaNaN@pyload.net"),
+        ("RaNaN", "RaNaN@pyload.org"),
```
Comment (vuolter) [inline src/pyload/plugins/addons/Captcha9Kw.py:53]: We use now pyload.net

File: src/pyload/plugins/addons/AntiVirus.py [hunk 2]
```diff
@@ -73,7 +73,10 @@ def scan(self, pyfile, thread):
         pyfile.set_progress(0)
 
         try:
-            p = subprocess.Popen([avfile, avargs, target])
+            p = Popen([avfile, avargs, target],
```
Comment (vuolter) [inline src/pyload/plugins/addons/AntiVirus.py:76]: We should keep `subprocess.Popen`

File: src/pyload/plugins/addons/Captcha9Kw.py (hunk 1 above)
Comment (OzzieIsaacs) [reply src/pyload/plugins/addons/Captcha9Kw.py]: It's changed to .net

File: src/pyload/plugins/addons/AntiVirus.py (hunk 2 above)
Comment (OzzieIsaacs) [reply src/pyload/plugins/addons/AntiVirus.py]: It's reverted

**Pull request closed without merging**